libp2p/daemon/daemonapi.nim

```diff
@@ -1306,3 +1306,20 @@ proc pubsubSubscribe*(api: DaemonAPI, topic: string,
   except Exception as exc:
     await api.closeConnection(transp)
     raise exc
+
+proc `$`*(pinfo: PeerInfo): string =
```

Comment:
hm, `$` is better off for roundtrippable stuff - for debug / logging output, better define a `shortLog` instead, else `$` gets picked up by serialization frameworks etc thinking they can use it